app/src/components/DataPanel.vue

```diff
@@ -94,13 +100,13 @@
             </v-row>
           </v-card>
           <v-card
-            v-else
+            v-else-if="!showMap || (showMap && indicatorObject.display.customAreaIndicator)"
             class="fill-height"
             :style="`height: ${$vuetify.breakpoint.mdAndUp ? (expanded
                               ? (bannerHeight ? 65 : 70) : 40) : 60}vh;`"
             ref="mapPanel"
           >
-            <full-screen-button />
+            <!-- <full-screen-button /> -->
```

Comment:
I think this removes the fullscreen button also for charts by mistake. On production, charts can be fullscreened, it would be nice to 1) keep this functionality and 2) also allow it for custom area charts if possible